tools/travis/build.sh

```diff
@@ -3,7 +3,7 @@
 
 SCRIPTDIR=$(cd $(dirname "$0") && pwd)
 ROOTDIR="$SCRIPTDIR/../.."
-WHISKDIR="$ROOTDIR/openwhisk"
+WHISKDIR="$ROOTDIR/../openwhisk"
```

Comment:
you need to change the location of where openwhisk is cloned to if you want to make this change.